libs/langchain/langchain/chains/query_constructor/parser.py

```diff
@@ -52,6 +54,11 @@ def v_args(*args: Any, **kwargs: Any) -> Any:  # type: ignore
 """
 
 
+class ISO8601Date(TypedDict):
```

Comment:
Perhaps a doc-string here to explain what the date format is so folks don't have to look up what iso-8601 is? (YYYY-MM-DD)